releasenotes/config.yaml

```diff
@@ -9,6 +9,8 @@ sections:
   - [api, API Changes]
   - [security, Security Issues]
   - [fixes, Bug Fixes]
+  - [other, Other Changes]
```

Comment:
does this order actually matter? https://ddtrace.readthedocs.io/en/stable/release_notes.html#v1-6-0 doesn't have the upgrade section above features 🤔

Reply:
oh my bad, i was looking at the latest commit, not the PR diff